main/index.html

```diff
@@ -45,16 +47,19 @@
                         <div class=" navbar-right">
                             <ul class="nav navbar-nav">
                                 <li>
-                                    <a id="nav_about" href="#about" class="nav-link">ABOUT</a>
+                                    <a id="nav_about" href="#about" class="nav-link"><strong>ABOUT</strong></a>
```

Comment:
idk why this is done. It was better not bold

Reply:
Ill change that back.